iroha-cli/interactive/impl/interactive_common_cli.cpp

```diff
@@ -123,9 +141,16 @@ namespace iroha_cli {
         std::for_each(params_description.value().begin(),
                       params_description.value().end(),
                       [&params](auto param) {
-                        auto val = promptString(param);
-                        if (val and not val.value().empty()) {
-                          params.push_back(val.value());
+                        auto val = promptString(*param);
```

Comment:
Prefer using [monadic bind operator](https://github.com/uditha-atukorala/iroha/blob/af57b48d245811ea48c61778833761e7eb5e5409/libs/common/types.hpp#L149-L175)
```cpp
promptString(*param) | [&](auto &val) {
  if (not val.value().empty()) {
  ...
};
```

Reply:
1. Do not forget to include "common/types.hpp"
2. The operator itself is in the namespace, so you may use `using namespace iroha;` inside the function

Reply:
Was missing `using namespace iroha;`. Thanks.